src/App.jsx

```diff
@@ -1,54 +1,17 @@
 import React from "react";
-import EmailVerify from "./component/email/EmailVerify";
-import Home from "./molecules/Home";
-import EmptyWishlistViewProducts from "./pages/EmptyWishlistViewProducts";
-import WishlistViewProducts from "./pages/WishlistViewProducts";
-import AddNewProductPage1 from './AddNewProduct/pageOne/AddNewProductPage1'
-import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+
```

Comment:
@modibboyusuph  You did not have to delete the previous code to fix the conflict.